terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/runners.test.ts

```diff
@@ -1173,7 +1273,87 @@ describe('createRunner', () => {
     });
 
     it('calls run instances with the correct config for repo && windows', async () => {
-      const runnerParameters = {
+      const runnerParameters: RunnerInputParameters = {
+        runnerConfig: runnerConfigFn,
+        environment: 'wg113',
+        repoName: 'SomeAwesomeCoder/some-amazing-library',
+        orgName: undefined,
+        runnerType: {
+          instance_type: 'c5.2xlarge',
+          os: 'windows',
+          max_available: 200,
+          disk_size: 100,
+          runnerTypeName: 'linuxCpu',
+          is_ephemeral: true,
+        },
+        repositoryOwner: 'SomeAwesomeCoder',
+        repositoryName: 'some-amazing-library',
+      };
+
+      await createRunner(runnerParameters, metrics);
+
+      expect(runnerConfigFn).toBeCalledTimes(1);
+      expect(runnerConfigFn).toBeCalledWith(config.awsRegion, false);
+      expect(mockEC2.runInstances).toHaveBeenCalledTimes(1);
+      const secGroup = Config.Instance.vpcIdToSecurityGroupIds.get('vpc-agdgaduwg113') || [];
+      expect(mockEC2.runInstances).toBeCalledWith({
+        MaxCount: 1,
+        MinCount: 1,
+        LaunchTemplate: {
+          LaunchTemplateName: Config.Instance.launchTemplateNameWindows,
+          Version: Config.Instance.launchTemplateVersionWindows,
+        },
+        InstanceType: runnerParameters.runnerType.instance_type,
+        BlockDeviceMappings: [
+          {
+            DeviceName: '/dev/sda1',
+            Ebs: {
+              VolumeSize: runnerParameters.runnerType.disk_size,
+              VolumeType: 'gp3',
+              Encrypted: true,
+              DeleteOnTermination: true,
+            },
+          },
+        ],
+        NetworkInterfaces: [
+          {
+            Ipv6AddressCount: 1,
+            AssociatePublicIpAddress: true,
+            SubnetId: 'sub-0113',
+            Groups: secGroup,
+            DeviceIndex: 0,
+          },
+        ],
+        TagSpecifications: [
+          {
+            ResourceType: 'instance',
+            Tags: [
+              { Key: 'Application', Value: 'github-action-runner' },
+              { Key: 'RunnerType', Value: runnerParameters.runnerType.runnerTypeName },
+              { Key: 'RepositoryOwner', Value: runnerParameters.repositoryOwner },
+              { Key: 'RepositoryName', Value: runnerParameters.repositoryName },
+              { Key: 'InstanceType', Value: runnerParameters.runnerType.instance_type },
+              {
+                Key: 'Repo',
+                Value: runnerParameters.repoName,
+              },
+            ],
+          },
+        ],
```

Comment:
in this test, only the tags that were passed in actually matter, right?  

In the interest of making the test more resilient to future changes, can you please change the expectation here to only check that those particular tags have the expected values and ignore everything else?